modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -219,6 +219,9 @@ public void processOpts() {
             importMapping.remove("DateTime");
             instantiationTypes.put("array", "java.util.ArrayList");
             instantiationTypes.put("map", "java.util.HashMap");
+        } else {
+            languageSpecificPrimitives.add("List");
+            languageSpecificPrimitives.add("Map");
```

Comment:
These 2 lines caused import error in the updated Java client.

To repeat the issue, run
```
./bin/java-petstore-all.sh
mvn verify -Psamples
```
Please commit the updated Java Petstore samples too.

(testing with C# client looks good. so far no issues found)